lib/objects/objectsInMemFileDB.js

```diff
@@ -2569,6 +2574,52 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
             this.writeTimer = null;
         }
     }
+
+    isEquivalentValue(a, b, ignoreField) {
+        // Create arrays of property names
+        if (a === null || a === undefined || b === null || b === undefined) {
+            return (a === b);
+        }
+        const aProps = Object.getOwnPropertyNames(a);
```

Comment:
Is there any case where the object has non-enumerable properties? In that case, Object.keys would exclude them. But I guess not.